code/modules/hydroponics/grown/cereals.dm

```diff
@@ -87,6 +90,7 @@
 	seed = /obj/item/seeds/wheat/meat
 	foodtype = MEAT | GRAIN
 	grind_results = list("flour" = 0, "blood" = 0)
+	tastes = list("meatwheat" = 1)
```

Comment:
nitpick: maybe `list("meat" = 1, "wheat" = 1)` instead?

Reply:
second this